STACpopulator/extensions/thredds.py

```diff
@@ -22,15 +22,19 @@ class THREDDSMetadata:
         ServiceType.opendap: pystac.MediaType.HTML,
         ServiceType.wcs: pystac.MediaType.XML,
         ServiceType.wms: pystac.MediaType.XML,
-        ServiceType.netcdfsubset: "application/x-netcdf",
+        ServiceType.netcdfsubset: "application/x-netcdf",  # used in THREDDS version < 5.0
+        ServiceType.netcdfsubsetgrid: "application/x-netcdf",  # used in THREDDS version > 5.0
```

Comment:
This adds support for the new services for netcdfsubset introduced in THREDDS version 5